File: Mar_PHP/Assignments/BasicPrograms/11.php
<!DOCTYPE html>
<html>
<head>
    <title>Sum of Digits</title>
</head>
<body>

<form method="post">
    <label for="number">Enter a number to calculate the sum of its digits:</label>
    <input type="number" id="number" name="number">
    <button type="submit">Calculate Sum</button>
</form>

<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
    $n = $_POST['number'];
    $sum = 0;
    $ori = $n;

    while ($n != 0) {
        $remainder = $n % 10;
        $sum += $remainder;
        $n = (int)($n / 10);
    }

    echo "The sum of digits of $ori is: $sum";
}
?>

</body>
</html>

File: Jan_Basic_Programming/Assignments/C/11.c
#include<stdio.h>
int main(){
    int a, b=0, sum=0; 
    printf("Enter Num : ");
    scanf("%d", &a);
    b=a;

    while(b>0){
        sum = sum + (b%10);
        b/=10;
    }
    printf("Sum : %d", sum);
 return 0;
}